### PR TITLE
Update WorkflowTitleValidator to validate titles in .icon file

### PR DESCRIPTION
**Pull request by r7-kszczepanskagorna**
## Description
Ticket [MC-62] 

## Testing
There are unit tests

### CODE REVIEW

**Review by jschipp-r7 — DISMISSED** [review]

**Review by mrinehart-r7 — APPROVED** [review]

**Comment by mrinehart-r7** [comment]
@r7-kszczepanskagorna approved pending fixing the failing unit tests

**Comment by jschipp-r7** [comment]
@r7-kszczepanskagorna can you fix the unit-tests, then we can merge - thanks!

**Review by jschipp-r7 — APPROVED** [review]

File: icon_validator/rules/workflow_validators/workflow_title_validator.py
Comment (pmara-r7) [inline icon_validator/rules/workflow_validators/workflow_title_validator.py:19]: Wouldn't minFraud and ifconfig.co trip the below exception?

Comment (mrinehart-r7) [reply icon_validator/rules/workflow_validators/workflow_title_validator.py:19]: This would trip that.

File: README.md
Comment (pmara-r7) [inline README.md:65]:
```suggestion
* 2.44.0 - Update WorkflowTitleValidator to validate titles in `.icon` file
```

File: icon_validator/rules/workflow_validators/workflow_title_validator.py
Comment (pmara-r7) [reply icon_validator/rules/workflow_validators/workflow_title_validator.py:19]: @blaxminarayan-r7 this comment was waiting responding and is the reason we did not merge this.

File: setup.py
Comment (pmara-r7) [inline setup.py:10]:
```suggestion
    version="2.44.0",
```

**Review by pmara-r7 — CHANGES_REQUESTED** [review]

**Comment by pmara-r7** [comment]
Merge conflicts can be fixed pending comment being addressed

File: icon_validator/rules/workflow_validators/workflow_title_validator.py
Comment (r7-mgorny) [reply icon_validator/rules/workflow_validators/workflow_title_validator.py:19]: @pmara-r7 We only add here more info to the Exception so when it raise we know exactly in which file the error occur. Could you please provide more info what you mean about `minFraud` and `ifconfig.co`?

Comment (pmara-r7) [reply icon_validator/rules/workflow_validators/workflow_title_validator.py:19]: We were confused about the comment in the code (I see that you are not changing that) but we were just concerned about what that line actually does. For this release, there is no need to focus closely on this if there is no bug raised around it in my opinion.

**Review by pmara-r7 — APPROVED** [review]

**Review by blaxminarayan-r7 — APPROVED** [review]

**Pull request merged**
